CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Added CHANGELOG.md (#10)
+- Added importmap support (#9)
+
+## [0.2.0] - 2023-10-07
+
+### Changed
+
+- Update HTMX lib to 1.9.2 (#4)
+- Update GH actions build
+
+## [0.1.0] - 2021-01-11
```

Comment:
I'd need some help from previous committers to complete and validate this previous changes. CC @JulianPasquale

Reply:
I think we can add some generic information, like:
- Initial release
- Introduce installation generator (support for Sprokets and Webpacker)